scripts/roots_of_quadratic.py

```diff
@@ -0,0 +1,8 @@
+#roots of quadratic equation in the form of ax^2 + bx + c = 0
+def roots(a,b,c):
+    x1 = (-b + ((b**2 - 4*a*c)**0.5))/(2*a)
+    x2 = (-b - ((b**2 - 4*a*c)**0.5))/(2*a)
+    print ("The roots are:", x1, " and ", x2)
+
+a,b,c = input("Enter the values of a, b, and c (seperated by commas) for the equation ax^2 + bx + c = 0 : ").split(',')
```

Comment:
Could you move line 7 and 8 inside a main function and call it using:
```
if __name__ == '__main__':
        main()
```
To stay consistent with the rest of the scripts in the repo?

Everything else looks good to me!